[2025-04-09T16:34:17.666+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-04-09T16:34:17.704+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: extract_raw_data_pipeline.extract_raw_data manual__2025-04-09T16:34:15.402601+00:00 [queued]>
[2025-04-09T16:34:17.719+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: extract_raw_data_pipeline.extract_raw_data manual__2025-04-09T16:34:15.402601+00:00 [queued]>
[2025-04-09T16:34:17.720+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2025-04-09T16:34:17.843+0000] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): extract_raw_data> on 2025-04-09 16:34:15.402601+00:00
[2025-04-09T16:34:17.850+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=1297) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-04-09T16:34:17.852+0000] {standard_task_runner.py:72} INFO - Started process 1301 to run task
[2025-04-09T16:34:17.852+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'extract_raw_data_pipeline', 'extract_raw_data', 'manual__2025-04-09T16:34:15.402601+00:00', '--job-id', '93', '--raw', '--subdir', 'DAGS_FOLDER/driven_data_pipeline.py', '--cfg-path', '/tmp/tmpt0w4_yfo']
[2025-04-09T16:34:17.853+0000] {standard_task_runner.py:105} INFO - Job 93: Subtask extract_raw_data
[2025-04-09T16:34:18.042+0000] {task_command.py:467} INFO - Running <TaskInstance: extract_raw_data_pipeline.extract_raw_data manual__2025-04-09T16:34:15.402601+00:00 [running]> on host 5a8aeda502d0
[2025-04-09T16:34:18.190+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='extract_raw_data_pipeline' AIRFLOW_CTX_TASK_ID='extract_raw_data' AIRFLOW_CTX_EXECUTION_DATE='2025-04-09T16:34:15.402601+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-04-09T16:34:15.402601+00:00'
[2025-04-09T16:34:18.191+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-04-09T16:34:18.216+0000] {driven_data_pipeline.py:147} INFO - Started batch processing for 2025-04-09.
[2025-04-09T16:34:18.216+0000] {driven_data_pipeline.py:34} INFO - Created synthetic data for RO country code.
[2025-04-09T16:34:18.655+0000] {driven_data_pipeline.py:104} INFO - Written 890 records to the CSV file.
[2025-04-09T16:34:19.556+0000] {driven_data_pipeline.py:120} INFO - Added UUID to the dataset.
[2025-04-09T16:34:19.558+0000] {driven_data_pipeline.py:155} INFO - Finished batch processing 2025-04-09.
[2025-04-09T16:34:19.558+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-04-09T16:34:19.570+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-04-09T16:34:19.571+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=extract_raw_data_pipeline, task_id=extract_raw_data, run_id=manual__2025-04-09T16:34:15.402601+00:00, execution_date=20250409T163415, start_date=20250409T163417, end_date=20250409T163419
[2025-04-09T16:34:19.638+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-04-09T16:34:19.665+0000] {taskinstance.py:3900} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-04-09T16:34:19.695+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
